src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -46,16 +48,22 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
   startWith(CallNotStarted, None)
 
   val call = key.scope
-  val tag = s"CallActor [UUID(${workflowDescriptor.shortId}):${key.tag}]"
+  val akkaLogger = Logging(context.system, classOf[CallActor])
+  val logger = WorkflowLogger(
```

Comment:
If I counted right this would be but 1 char beyond the line length czar's preferred limit. I suspect he'd be cool with bending the rule in this case.